diesel/src/query_dsl/distinct_dsl.rs

```diff
@@ -40,10 +39,10 @@ where
 /// [`QueryDsl`]: ../trait.QueryDsl.html
 #[cfg(feature = "postgres")]
 pub trait DistinctOnDsl<Selection> {
-    /// Query with `DISTINCT ON()` added
+    /// The type returned by `.distinct_on`
     type Output;
 
-    /// Should return query with `DISTINCT ON` added
+    /// See the trait documentation
```

Comment:
This is pretty non-documentary. If this is what you really want people to do, at least include a link

Reply:
I would be linking to the page they are currently on.